src/main/java/com/basho/riak/client/core/query/timeseries/TableDefinition.java

```diff
@@ -0,0 +1,85 @@
+package com.basho.riak.client.core.query.timeseries;
+
+import java.util.List;
+
+/**
+ * Holds a definition for a Table in Time Series Riak.
+ * Immutable once created.
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.4
+ */
+public class TableDefinition
+{
+    private final String tableName;
+    private final List<FullColumnDescription> fullColumnDescriptions;
```

Comment:
It might be a good idea to avoid field name duplication on this level, we have two options:
  - Change FullColumnDescription hashCode/equals to use only Field Name and store descriptions in HashSet
 - Store descriptions in a HashMap and use FieldName as a key